### PR TITLE
Add GitHub issue templates for a feature request and a help request (closes ##961)

### PR DESCRIPTION
**Pull request by mkondratek**

### CODE REVIEW

**Review by PawelLipski — APPROVED** [review]

File: .github/ISSUE_TEMPLATE/feature_request.md [hunk 1]
```diff
@@ -0,0 +1,11 @@
+---
+name: Feature request
+about: Request a new feature to the plugin
+title: ''
+labels: feature
```
Comment (mkondratek) [inline .github/ISSUE_TEMPLATE/feature_request.md:5]: Or better `externally requested`? 🤔 I guess sometime we will be those who request

File: .github/ISSUE_TEMPLATE/feature_request.md [hunk 2]
```diff
@@ -0,0 +1,11 @@
...
+assignees: ''
+
+---
+
+## Feature description
+<!--- Describe your idea. --->
```
Comment (mkondratek) [inline .github/ISSUE_TEMPLATE/feature_request.md:11]: Do we want to use comments? The final issue should look cleaner. I have no strong feelings. We could use them in the bug report template too. wdyt?

File: .github/ISSUE_TEMPLATE/feature_request.md [hunk 3]
```diff
@@ -0,0 +1,11 @@
+---
+name: Feature request
```
Comment (mkondratek) [inline .github/ISSUE_TEMPLATE/feature_request.md:2]: Can I test it somehow? Or it must be deployed (to master I guess as it is the default branch)?

File: .github/ISSUE_TEMPLATE/feature_request.md (hunk 1 above)
Comment (PawelLipski) [reply .github/ISSUE_TEMPLATE/feature_request.md:5]: +1, why not, we can removed the label then

File: .github/ISSUE_TEMPLATE/feature_request.md (hunk 2 above)
Comment (PawelLipski) [reply .github/ISSUE_TEMPLATE/feature_request.md:11]: +1

Comment (PawelLipski) [reply .github/ISSUE_TEMPLATE/feature_request.md:11]: Pls update bug template indeed

Comment (PawelLipski) [reply .github/ISSUE_TEMPLATE/feature_request.md:11]: Oh, there's even #1380 for that

File: .github/ISSUE_TEMPLATE/feature_request.md (hunk 1 above)
Comment (mkondratek) [reply .github/ISSUE_TEMPLATE/feature_request.md:5]: yup, I think `externally requested` looks more premium Xd

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 4]
```diff
@@ -8,18 +8,18 @@ assignees: ''
 ---
 
 ## Bug description
-Please include steps to reproduce (like `go to...`/`click on...` etc.) + expected and actual behavior.
+<!--- Please include steps to reproduce (like `go to...`/`click on...` etc.) + expected and actual behavior.
```
Comment (PawelLipski) [inline .github/ISSUE_TEMPLATE/bug_report.md:11]: Triple, not double hyphen? 🤔 not an error probably tho

File: .github/ISSUE_TEMPLATE/feature_request.md [hunk 5]
```diff
@@ -0,0 +1,11 @@
...
+about: Request a new feature to the plugin
+title: ''
+labels: "externally requested"
```
[the remark this answers is not in the document]
Comment (PawelLipski) [reply .github/ISSUE_TEMPLATE/feature_request.md:5]: Maybe
```suggestion
labels: ["externally requested", "feature"]
```
Judging by
image

File: .github/ISSUE_TEMPLATE/help_or_support.md [hunk 6]
```diff
@@ -0,0 +1,15 @@
+---
+name: Help/Support
+about: Ask a question or request support about the plugin
+title: ''
+labels: question
```
Comment (PawelLipski) [inline .github/ISSUE_TEMPLATE/help_or_support.md:5]: I've just added this label to this repo

Comment (mkondratek) [reply .github/ISSUE_TEMPLATE/help_or_support.md:5]: no, I did that!

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 7]
```diff
@@ -8,18 +8,18 @@ assignees: ''
 ---
 
 ## Bug description
-Please include steps to reproduce (like `go to...`/`click on...` etc.) + expected and actual behavior.
+<!-- Please include steps to reproduce (like `go to...`/`click on...` etc.) + expected and actual behavior.
```
[the remark this answers is not in the document]
Comment (PawelLipski) [reply .github/ISSUE_TEMPLATE/bug_report.md:11]: yep

**Comment by PawelLipski** [comment]
I've changed the default branch to `develop` for a moment to see the results:

image

image

image

All good!

**Pull request merged**